src/iframe-attributes.js

```diff
@@ -57,6 +60,14 @@ export function getContextMetadata(
       href: locationHref,
     },
     startTime,
+    tagName: element.tagName,
+    mode: getModeObject(),
+    canary: isCanary(parentWindow),
+    hidden: !viewer.isVisible(),
+    initialIntersection: element.getIntersectionChangeEntry ?
```

Comment:
Yes, this method does not exist for the A4A ad element. Without this check, it fails in the A4A cases.